src/components/molecules/AssetStatus.tsx

```diff
@@ -0,0 +1,46 @@
+import { DDO, DID } from '@oceanprotocol/lib'
+import axios, { CancelTokenSource } from 'axios'
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useOcean } from '../../providers/Ocean'
+import { isFileValid } from '../../utils/provider'
+import Status from '../atoms/Status'
+import Tooltip from '../atoms/Tooltip'
+import styles from './AssetStatus.module.css'
+
+export default function AssetStatus({ ddo }: { ddo: DDO | DID }): ReactElement {
+  const { ocean } = useOcean()
+  const [fileConnectivity, setFileConnectivity] = useState<boolean>(true)
```

Comment:
just small naming thing, would be more clear to name these `isFileValid, setFileValid` to stay consistent with the used method and to better signal the type of variable to other developers (`is` prefix always implies a boolean)

Reply:
We can't use `isFileValid` as it is colliding with the method name from `src/utils/provider.ts`.

Reply:
good point, didn't see that. State var definitely should be called `isFileValid` but contradicting my own first argument the method then could be like `checkFileValidity()` but sounds kinda weird, maybe just go with `checkFile()`. Naming is so hard 😄